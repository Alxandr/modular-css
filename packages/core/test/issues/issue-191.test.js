"use strict";

const fs = require("fs");
    
const namer = require("test-utils/namer.js");
const { find, temp } = require("test-utils/fixtures.js");

const Processor = require("../../processor.js");

describe("/issues", () => {
    describe("/191", () => {
        var fn = it;
        
        // Verify that filesystem is case-insensitive before bothering
        fs.writeFileSync(temp("sensitive.txt"));

        try {
            fs.statSync(temp("SENSITIVE.txt"));
        } catch(e) {
            fn = it.skip;
        }

        fn("should ignore case differences in file paths", async () => {
            const processor = new Processor({
                namer,
            });
            
            await processor.file(find("191.css"));

            const result = await processor.output();
            
            expect(result.css).toMatchSnapshot();
        });
    });
});
