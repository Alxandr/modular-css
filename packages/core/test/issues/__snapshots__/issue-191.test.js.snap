// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/issues /191 should ignore case differences in file paths 1`] = `
"/* fixtures/191/values.css */
.fooga {
    color: #F00;
}
/* fixtures/191/start.css */
"
`;
